featuretools/entityset/entityset.py

```diff
@@ -898,11 +898,13 @@ def add_last_time_indexes(self, updated_dataframes=None):
         """
         Calculates the last time index values for each dataframe (the last time
         an instance or children of that instance were observed).  Used when
-        calculating features using training windows
+        calculating features using training windows. Adds the last time index as
+        a series named _ft_last_time on the dataframe.
         Args:
             updated_dataframes (list[str]): List of dataframe names to update last_time_index for
                 (will update all parents of those dataframes as well)
         """
+        LTI_COLUMN_NAME = '_ft_last_time'
```

Comment:
I like this change - easy to update in the future if we need to! What do you think about moving this out of the method and also using this same variable in our test assertions?